app/src/main/kotlin/com/wire/android/ui/home/conversations/model/MessageTypes.kt

```diff
@@ -146,15 +149,14 @@ fun MessageButtonsContent(
 @SuppressLint("RememberReturnType")
 @Composable
 fun MessageButtonItem(
-    messageId: String,
     button: MessageButton,
     pendingButtonId: MessageButtonId?,
-    onClick: ((messageId: String, buttonId: String) -> Unit)?
+    onClick: ((buttonId: String) -> Unit)?
```

Comment:
I do no think this needs to be nullabe anymore ? Also I think it would be "cleaner" if we use here Clickable